target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
I understand that and child nodes inherit this pinctrl also, I bet there is a hidden meaning for that :)
Well, you're the boss atm and if you feel that it's really wrong then maybe we could please @heinzek to move rgmii and mdio pins to its places in r7800 dts while we are at it?